packages/atclient/src/monitor.c

```diff
@@ -39,6 +38,7 @@ void atclient_monitor_message_free(atclient_monitor_message *message) {
   case ATCLIENT_MONITOR_ERROR_DECRYPT_NOTIFICATION:
     atclient_atnotification_free(message->notification);
     free(message->notification);
+    __attribute__((fallthrough));
```

Comment:
Tells the compiler that this is an explicit fallthrough not an implicit one where a `break` is missing.